docs/DafnyRef/Attributes.md

```diff
@@ -147,7 +147,7 @@ It means that the post-condition may be assumed to be true
 without proof. In that case also the body of the function or
 method may be omitted.
 
-The `{:axiom}` attribute is also used for generated `reveal_*`
+The `{:axiom}` attribute is internally equivalent to `{:verify false}` and is also used for the internal `reveal_*`
```

Comment:
What's the difference between equivalent and internally equivalent?

I think any use of the word `internal` in documentation to refer to something not so important to the user, is a smell. I'd say either the user needs to know and then it's not internal, or they don't and then it shouldn't be mentioned in the documentation.

Reply:
There is a user-facing difference between something being an `{:axiom}`, meaning it's assumed to be true and will be axiom forever (and usually functions or methods annotated with `{:axiom}` do not have body) , vs. something annotated with `{:verify false}`, which looks like a temporary modifier to instruct Dafny to not proofread an existing body. Typically, in an IDE, we would emit warnings for `{:verify false}` but not for `{:axiom}`.
However, at the end, they have exactly the same semantics, and I was actually surprised to see they were the first time I saw that, which I think is brilliant. That's what I mean by "internally equivalent".
If this helps, you can think of an analogy with comments: Double slash is the same as triple slash for the code (it's code documentation), but triple slash means documentation that is typically API documentation, whereas double slash means it's for developers maintaining the code. In a concrete AST, I wouldn't be surprised if the two comment nodes are represented with the same structure, but with a flag.

So, in short, I believe that the first "internally equivalent" is appropriate ("Oh by the way, user, don't expect any difference from these two attributes from Dafny point of view, but of course use them as you think appropriate").

I'll remove the second one however because since we now hide the implementation of reveal_* as a lemma we shouldn't mention it in the documentation.